Cargo.toml

```diff
@@ -1,40 +1,39 @@
 [package]
 name = "logdna-client"
-version = "0.4.0"
+version = "0.5.0"
 authors = ["engineering@logdna.com"]
 edition = "2018"
 license = "MIT"
 description = "wrapper around LogDNA's Ingest API"
 
 [dependencies]
 #error handling
-quick-error = "1"
 thiserror = "1"
 
 #io
-bytes = "0.5"
-hyper = "0.13"
-tokio = "0.2"
+bytes = "1"
+hyper = { version = "0.14", features = ["client", "tcp", "http1"] }
+tokio = "1"
 http = "0.2"
 async-compression = {version = "0.3", features = ["futures-io", "gzip"]}
 
 # async
 futures = "0.3"
 async-trait = "0.1"
-async-buf-pool =  { git= "https://github.com:/logdna/async-buf-pool-rs.git", tag="0.2.0", version = "0.2" }
+async-buf-pool =  { git= "https://github.com:/logdna/async-buf-pool-rs.git", tag="0.3.0", version = "0.3"}
```

Comment:
Do you prefer to pin it to the tag?

branch `0.3.x` should allow us to update more easily

Reply:
I don't have any functionality changes planned for the buf pool at the moment, I think it's basically "done" as far as features for this lib is concerned, but I'll switch it to the branch so that this repo's CI can keep us honest about any changes we do end up making :)